tbx/project_styleguide/templates/patterns/navigation/components/primary-nav.html

```diff
@@ -1,9 +1,16 @@
 {% load wagtailcore_tags %}
 
+{% comment %}
+The `child_display` has 3 possible values:
+- hide_children - do not show a subnavigation
+- show_up_to_level1 - show level one child pages
+- show_up_to_level2 - show level one and level two child pages
+{% endcomment %}
+
 {% for item in primarynav %}
     {% with children=item.value.page.get_children.live.public.in_menu child_display=item.value.child_display_behaviour %}
         <li class="primary-nav-desktop__item {% if children and child_display != 'hide_children' %}primary-nav-desktop__item--is-parent{% endif %}"
-            {% if children and not child_display != 'hide_children' %}data-has-subnav{% endif %}
+            {% if children and child_display != 'hide_children' %}data-has-subnav{% endif %}
```

Comment:
@engineervix this was another catch I found where we had slightly incorrect code - so this may be wrong on staging at the moment.